### PR TITLE
JTE/PKFE-46

### PR DESCRIPTION
**Pull request by justinnas**
Added file export functionality.

### CODE REVIEW

File: app/back-end/src/routes/workspace_route.py [hunk 1]
```diff
@@ -1168,3 +1169,75 @@ def import_file(relative_path=None):
...
+            uuid,
+            sid,
+        )
+
+        response = send_file(file_path, as_attachment=True)
+        
+        return response
+
+    except FileNotFoundError as e:
+        logger.error("FileNotFoundError: %s while accessing %s", e, user_workspace_dir)
```
Comment (mantvydasdeltuva) [inline app/back-end/src/routes/workspace_route.py:1204]: edit everywhere where logger and console feedback: `accessing -> exporting`

**Review by mantvydasdeltuva — CHANGES_REQUESTED** [review]
Great work! Minor oversights

File: app/front-end/src/features/editor/components/fileTreeView/fileTreeItem/fileTreeItemContextMenu.tsx [hunk 2]
```diff
@@ -158,6 +157,30 @@ export const FileTreeItemContextMenu: React.FC<FileTreeItemContextMenuProps> = (
     filesHistoryStateUpdate(undefined, { id: item.id, label: item.label, type: item.fileType || FileTypes.FILE });
   };
 
+  const handleExport = async () => {
```
[the remark this answers is not in the document]
Comment (justinnas) [reply app/front-end/src/features/editor/components/fileTreeView/fileTreeItem/fileTreeItemContextMenu.tsx:160]: Good idea

File: app/back-end/src/routes/workspace_route.py (hunk 1 above)
Comment (justinnas) [reply app/back-end/src/routes/workspace_route.py:1204]: Great catch

**Review by mantvydasdeltuva — APPROVED** [review]
Great work!

**Pull request merged**